src/create-dashboard.js

```diff
@@ -75,6 +77,10 @@ module.exports = function (selector, vizJSON, opts) {
 
   dashboardView.render();
 
+  vis.instantiateMap({
+    invalidateMapSize: true
```

Comment:
This method call doesn't [match the implementation](https://github.com/CartoDB/cartodb.js/pull/1104#discussion-diff-53049654) (an object instead of a boolean).

But even so, like I [commented on the other PR](https://github.com/CartoDB/cartodb.js/pull/1104#discussion-diff-53049654), why having to pass/know about the "invalidate map size" detail here?